functions.php

```diff
@@ -442,9 +442,14 @@ function odm_return_all_topics( $query ) {
 
 function odm_search_pre_get_posts($query)
 {
+  if(get_post_type()){
+    $post_type = $query->query['post_type'][0];
```

Comment:
@Huyeng What is the variable $post_type used for? I do not see any reference to it on this function...